src/applications/disability-benefits/2346/api/index.js

```diff
@@ -0,0 +1,34 @@
+import { apiRequest } from '../../../../platform/utilities/api';
+
+export const getDLCBatteryDataApi = () => apiRequest('/dlc/orderstatus/1014/1');
+// NOTE: Decision was to take socks out of MVP -@maharielrosario at 1/28/2020, 9:49:20 AM
+// export const getDLCSocksDataApi = () => apiRequest('/dlc/orderstatus/1014/2');
+export const getDLCAccessoriesDataApi = () =>
+  apiRequest('/dlc/orderstatus/1014/3');
+export const getVeteranInformation = () =>
+  apiRequest(`/in_progress_forms/MDOT`, {
```

Comment:
Also here

Reply:
@bradl3yC  We have` async/await` being called in our actions creators. We just place those `api` calls in a separate file so that if/when this application grows we can separate the `actions` files and still call the endpoints from one central location. So we are using `async/await` in `src/applications/disability-benefits/2346/actions/index.js` 
```
export const getVeteranAddressInfo = () => async dispatch => {
  try {
    const data = await getVeteranInformation();
    const veteranAddresses = {
      veteranAddress: data.formData.veteranAddress,
      email: data.formData.email,
    };
    dispatch(fetchVeteranAddressSuccess(veteranAddresses));
  } catch (error) {
    dispatch(
      fetchVeteranAddressFailure(error, 'failed to retrieve data from api'),
    );
  }
};
```

Reply:
👍 sounds good

Reply:
To clarify, the `getDLCAccessoriesDataApi` returns a promise because `apiRequest` returns a promise, so that promise is awaited on the line where we do:
```js
const data = await getDLCAccessoriesDataApi();
```

If they were to add more loc to `getDLCAccessoriesDataApi`, they would need to make that function `async await` and `await apiRequest('/dlc/orderstatus/1014/3')`

Reply:
Unrelated but the variable naming here could certainly use a little love to aid readability 🙂

Reply:
@kelsonic The `getDLCAccessoriesDataApi ` has been deleted. That was a mock end point we don't have anymore.

Reply:
@kelsonic Anything with `dlc` we will rename once we implement thsi functionality.  But right now all `dlc` related anything has been removed.